packages/ui-library/src/components/button-group/index.stories.ts

```diff
@@ -104,10 +104,7 @@ export const ButtonGroup = (
     })}
   `;
 
-  return html`<div class="wrapper">
-    ${BlrButtonGroupRenderFunction(params, contentButtons)}
-    <div class="wrapper"></div>
-  </div>`;
+  return html` <div class="wrapper">${BlrButtonGroupRenderFunction(params, contentButtons)}</div>`;
```

Comment:
Just a note: The extra space between html and div, can it be removed? Since the space in inside template literal, the extra space might get rendered as well but not sure.